### PR TITLE
Draft edits for Schema.org and RRM transformations

### PR DESCRIPTION
**Pull request by sohkai**
Should be reviewed last, and will likely be split up into two parts so the pretty diff can work.

TODO: 

* [x] Rebase on top of all other changes
* [x] grep for `data-structure/` or `schema/` and update as necessary with #13 
* [x] Run through spell check again

### CODE REVIEW

File: README.md
Comment (TimDaub) [inline README.md]: Both Rule 5 and 6 don't match the framing in the upper section #the-lccs-ten-targets.

My suggestion would be to either:

- copy exact framing from above
- remove repetition of rules

File: README.md
Comment (TimDaub) [inline README.md]: We removed "Licenses are usually intended to be read by humans".
I think this statement should still be included somehow.

**Review by TimDaub — APPROVED** [review]

File: README.md
Comment (sohkai) [reply README.md]: Nice catch; I've opted to copy the framing from above. See f8dbfb5934d53967fab2d3c41e1e021b576e9436.

File: README.md
Comment (sohkai) [reply README.md]: See f8dbfb5934d53967fab2d3c41e1e021b576e9436.

**Comment by sohkai** [comment]
@TimDaub Not sure if I left this unmerged before I left, or if it was approved after, but I fixed the merge conflict and I'll leave you to choose what to do with it 💃 :).

**Pull request merged**
